src/Simulation/qdk_sim_rs/README.md

```diff
@@ -0,0 +1,41 @@
+# Quantum Development Kit Experimental Simulators
```

Comment:
Can you please add some usage examples - how will someone call the simulator and get results from Python, C# and C++

Reply:
Definitely agreed, but this may not be the right place for C# examples in particular — this file is the README specifically for the native library portion of the experimental simulators, and the C and Python APIs built from that library (notably the API for calling directly into the simulator, and not the Python + Q# API covered by https://github.com/microsoft/iqsharp/pull/449). Following that, this README.md is formatted for use with rustdoc / https://docs.rs. Will look for appropriate places to add such examples.

Reply:
Added significantly to the Rust documentation generated by `rustdoc` or https://docs.rs, and added new .md files explaining the C# bindings. Will add docs on Python +Q# usage (as opposed to the Python-only usage documented here) to the IQ# repo.